maintain-tidb-using-tiup.md

```diff
@@ -136,6 +136,50 @@ server_configs:
 
 Then run the `tiup cluster reload ${cluster-name} -N tidb` command to rolling restart the TiDB component.
 
+## Replace with a hotfix package
```

Comment:
This section is duplicate with https://pingcap.com/docs/dev/tiup/tiup-cluster/#update-component, so the review is a no-brainer. @lilin90 FYI